core/src/main/java/org/apache/accumulo/core/client/ScannerBase.java

```diff
@@ -242,4 +242,30 @@
    * @since 1.8.0
    */
   long getBatchTimeout(TimeUnit timeUnit);
+
+  /**
+   * Sets the name of the classloader context on this scanner
```

Comment:
We could make the pointer a little more stable by using Accumulo version #. Like saying see section X in the 1.8.0 user manual.